### DISCUSSION TITLE
BUILD FAILED (macOS 12.7.3 on x86_64 using ruby-build 20240119)

### DISCUSSION THREAD

**Discussion by banggibima** [rbenv/ruby-build#2345] [Build failures] [answered]
I've tried to install ruby 3.30 through rbenv with the command: `rbenv install 3.3.0`
I get an error like this. even though I have applied the method that has been exemplified in the installing guide on gorails.com

![Screen Shot 2024-02-08 at 10 41 12](https://github.com/rbenv/ruby-build/assets/38267431/560c5ae5-f6c9-45ed-8a4f-877566a35702)

**Comment by mislav** [comment, marked as answer]
Hi, it looks like the `configure` step failed, but there is no error message in this output. To check what happened, you will need to inspect the build log. Please observe the yellow highlighted line. If you find what looks like an error message, please share it here.

**Reply by banggibima** [reply]
Thank you for your response and help. It has been resolved. It turns out the cause of this error was because I hadn't installed Xcode Command Line Tools
